packages/scandipwa/src/route/UrlRewrites/UrlRewrites.component.js

```diff
@@ -73,14 +73,16 @@ export class UrlRewrites extends PureComponent {
         switch (type) {
         case TYPE_PRODUCT:
             return (
-                <ProductPage
-                  history={ history }
-                  location={ location }
-                  match={ match }
-                  productSKU={ productSKU }
-                  productID={ id }
-                  key={ id }
-                />
+                productSKU ? (
+                    <ProductPage
+                      history={ history }
+                      location={ location }
+                      match={ match }
+                      productSKU={ productSKU }
+                      productID={ id }
+                      key={ id }
+                    />
+                ) : this.renderDefaultPage()
```

Comment:
Please put this block to a separate render method so it would be convenient to override it if necessary.